### PR TITLE
Localize app

### PR DESCRIPTION
**Pull request by ebrahim-elgaml**
Connected to #12

Localize all the pages with ember-i18n
Import moment locale
add translation for English and German

Note: Check German translation as it is all based on google translate

### CODE REVIEW

File: app/instance-initializers/i18n.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    });
+  }
+};
+
+function determineLocale(availableLocales) {
+  let userLanguages = navigator.languages || [];
+  userLanguages.push(navigator.language, navigator.userLanguage, 'en');
+
+  return userLanguages.compact().find( language => {
+    return availableLocales.indexOf(language) > -1 ||
```
Comment (skddc) [inline app/instance-initializers/i18n.js]: I think we can use [`Array.prototype.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes?v=control) here.

**Comment by skddc** [comment]
@ebrahim-elgaml I added one suggestion about improving the code, but it all LGTM in general. Good job!

**Comment by ebrahim-elgaml** [comment]
@skddc Please check again now

**Comment by skddc** [comment]
@ebrahim-elgaml Did you actually try this out in multiple browsers? The `navigator` APIs are inconsistent between browsers oftentimes, and in this case, the language one seems to not work in Firefox as it is now:

> TypeError: can't define array index property past the end of an array with non-writable length

From this line:

```js
userLanguages.push(navigator.language, navigator.userLanguage, 'en');
```

**Comment by skddc** [comment]
Ping @ebrahim-elgaml

**Comment by ebrahim-elgaml** [comment]
@skddc
Sorry I have not seen that comment before.
I have tested it only on chrome. I will work on that issue

**Comment by skddc** [comment]
Great, thanks!

**Comment by ebrahim-elgaml** [comment]
@skddc please check now

**Comment by skddc** [comment]
It'd be nice if your commit message (or at least the GitHub comment) would contain an explanation of how your change fixes the issue (i.e. what the issue actually was and why it's fixed now).

Otherwise I have to assume things instead of knowing for sure what the intent of the code is.

**Comment by ebrahim-elgaml** [comment]
@skddc I have pushed again. Sorry for the commit description Please check again now

**Comment by skddc** [comment]
Thanks!

**Pull request merged**